lua/lspconfig/util.lua

```diff
@@ -378,6 +377,11 @@ function M.path.exists(filename)
   return stat and stat.type or false
 end
 
+--- @deprecated
+function M.path.escape_wildcards(path)
+  return path:gsub('([%[%]%?%*])', '\\%1')
```

Comment:
should this use the local one from line 214?

Reply:
> we could try removing this , I doubt anyone else is using it

Yeah, this is barely used when I did code search.

Reply:
OK, removing it instead.